APIs/ReleaseNotes.qmd

```diff
@@ -5,8 +5,105 @@ title: Release notes
 
 This page will include the list of changes made to Catalog APIs for each release.
 
-:::{.callout-note}
+# OData Catalog API updates
 
-More details will be added in the near future.
+## [2023-10-02] OData API single product response update
+
+To be compliant with the OData PRIP interfaces the **OData API** response for a single product query was updated. The response now returns the product itself instead of a list containing the product (assigned to the field `value`). The example reflecting the change is presented below.
```

Comment:
```suggestion
To comply with the OData PRIP interfaces, the **OData API** response for a single product query was updated. The response returns the product itself instead of a list containing the product (assigned to the field `value`). The example reflecting the change is presented below:
```